packages/calcite-components/src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -580,6 +581,14 @@ export class DatePickerMonth extends LitElement {
     const curMonDays = this.getCurrentMonthDays(month, year);
     const prevMonDays = this.getPreviousMonthDays(month, year, startOfWeek);
     const nextMonDays = this.getNextMonthDays(month, year, startOfWeek);
+    const numDaysDisplayed = curMonDays.length + prevMonDays.length + nextMonDays.length;
+    const numDaysToDisplay = NUM_WEEKS_TO_DISPLAY * DAYS_PER_WEEK;
```

Comment:
Nitpick:  numDaysToDisplay can be a const value `42`. Don't think we need to compute it every render.